BuildForSDG/EduKolab-Frontend
# create-react-app

**What the PR achieves** 
This PR does the following:

- Integrates the **create-react-app** setup with the existing **js-template** setup without breaking any of the existing testing or linting functions

- Adds a feature that checks for linting && test errors before starting, building, or deploying the app: locally or online.
It also runs this check before committing to Git.
If any lint or test errors are found, the following start, build, deploy or commit process is prevented


**Steps to test the PR locally**
1. Create a linting or test error on /src/app.js and run any of the following:

- ` npm start`
- `npm run build`
- `npm run deploy`


2. Make changes that creates an error and try to commit the change

**An npm error is thrown if an error exists inside the `/src` for the above 2 steps**

The check can be done solely by running `npm run check` 

Closes #9
